src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/Instrumentation.cs

```diff
@@ -43,14 +45,38 @@ public static void Initialize()
                         .AddSource("OpenTelemetry.AutoInstrumentation.*");
 
                     _tracerProvider = builder.Build();
-                    Console.WriteLine($">>>>>>>>>>>>>>>>>>>>>>> Process: {p.ProcessName}({p.Id}), initialized");
+                    Log("OpenTelemetry SDK loaded.");
                 }
             }
-            catch (Exception)
+            catch (Exception ex)
             {
-                // TODO: Should we have our own logger like datadog has?
+                Log($"OpenTelemetry SDK load exception: {ex}");
```

Comment:
do we need these `try/catches` at all?
the expcetion is thrown and should crash the application, right?

Reply:
Right, but I think it makes it easier to pinpoint any issue: the code is small but a few minutes here and there accumulate...